deployment/terraform/container_service.tf

```
@@ -120,6 +122,7 @@ resource "aws_ecs_task_definition" "app" {
   cpu                      = "${var.fargate_cpu}"
   memory                   = "${var.fargate_memory}"
 
+  task_role_arn      = "${aws_iam_role.ecs_task_role.arn}"
```

See justification for this suggestion where the resource is defined below.

```suggestion
  task_role_arn      = "${aws_iam_role.app_task_role.arn}"
```